fe/fe-core/src/main/java/com/starrocks/connector/ConnectorScanRangeMgr.java

```diff
@@ -0,0 +1,17 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.connector;
+
+import com.starrocks.thrift.TScanRangeLocations;
+
+public interface ConnectorScanRangeMgr {
```

Comment:
Better to give some comment to explain what this interface do. If you provide a example it will be better.